src/website/.env.sample

```
@@ -5,9 +5,14 @@ OPENCAGE_API_KEY=
 SLACK_WEBHOOK_URL=
 SLACK_CHANNEL=
 GOOGLE_SITE_VERIFICATION=
+# Shared PIN required to remove a photo from the /selfies conference wall.
+# Leave unset locally to skip the PIN check during development.
+CLEAN_AIR_FORUM_WALL_PIN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLEAN_AIR_FORUM_WALL_PIN key should go before the GOOGLE_SITE_VERIFICATION key

(UnorderedKey)